test/unit/operations/find.test.ts

```diff
@@ -43,7 +43,7 @@ describe('FindOperation', function () {
       it('should build basic find command with filter', async () => {
         const findOperation = new FindOperation(undefined, namespace, filter);
         const stub = sinon.stub(server, 'command').yieldsRight();
-        await promisify(findOperation.execute.bind(findOperation))(server, undefined);
+        await promisify(findOperation.executeCallback.bind(findOperation))(server, undefined);
```

Comment:
You can get rid of the `promisify` here instead of replacing `execute` with `executeCallback`. Remember that what `promisify` does is take an asynchronous function that returns its results via callback and changes it into one that returns a promise instead. That's exactly what you did with `execute` itself, so you can just await that directly.

The same goes for all the other unit tests like this below.